collect_app/src/main/java/org/odk/collect/android/utilities/ResetUtility.java

```diff
@@ -90,6 +94,11 @@ private void resetPreferences(Context context) {
         Collect.getInstance().initProperties();
     }
 
+    private void resetSMSSubmissionsHistory(Context context) {
+        new SmsSubmissionManager(context).clearSubmissions();
```

Comment:
Since I added this component I have been using Dagger to inject just in case it ever needs to be replaced with another implementation so we could probably inject it here if there's no issue with that. 😄